content/starter-configs/tutorials/wordpress.md

```diff
@@ -1,44 +1,46 @@
 ---
 title: "Wordpress"
-date: 2019-09-19T11:00:57-04:00
+date: 2023-06-22T13:28:00-06:00
 weight: 4
 ---
 
 Wondering how to configure Tugboat for a standard WordPress repository? Every site tends to have slightly different
 requirements, so you may need to do more customizing, but this should get you started.
 
-## Configure WordPress
+## First Things First
 
-For WordPress to use environment-specific settings, you'll need to make some changes to `wp-config.php`. One way to do
-this is to commit a "default" version of the file to the git repository, and include a "local" config file with override
-options. That local file should not be included in the git repository.
+We'll be following best practices of keeping Wordpress Core in a separate directory within the repo (for example,
+`/docroot`). If you have Wordpress Core files at the root of your directory, you'll need to restructure your repo to
+look more like this:
```

Comment:
> If you have Wordpress Core files at the root of your directory, you'll need to restructure your repo to look more like this:

If folks have Wordpress Core at the root, it's probably a requirement of their hosting provider. I think we should instead say something like "If you have Wordpress Core files at the root of your directory, you'll have to adapt this config to accommodate this. Feel free to reach out to support if you need help!" Or something?

Reply:
That makes sense.  I can doctor that up to support either option.

Reply:
That makes sense.  I can doctor that up to support either option.